src/app/tide/peptide_mods3.cc

```diff
@@ -214,9 +226,9 @@ class ModsOutputter {//: public IModsOutputter {
         int poss_max_ct = mod_table_->PossMaxCt(*aa, i, NTPEP);
         if (max_counts_[poss_max_ct] == 0) {
           int delta_index = mod_table_->PossDeltIx(*aa, i, NTPEP);
-          peptide_->add_modifications(mod_table_->EncodeMod(pos, delta_index));
-          OutputMods(1, counts);
-          peptide_->mutable_modifications()->RemoveLast();
+          peptide_->set_nterm_mod(mod_table_->EncodeMod(pos, delta_index));
+          OutputMods(0, counts);
```

Comment:
change from OutputMods(1, counts) -> OutputMods(0, counts)? It is worth double checking if it's an expected change